.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```

Comment:
Will this build the SDK once per day? If so, I'd not bother. Provided it builds master when it's updated, that should be fine. (What do we gain from daily builds?)

Reply:
I see an answer to this elsewhere!